tests/integration_tests/modules/test_users_groups.py

```diff
@@ -69,7 +69,10 @@ class TestUsersGroups:
     def test_users_groups(self, regex, getent_args, class_client):
         """Use getent to interrogate the various expected outcomes"""
         result = class_client.execute(["getent"] + getent_args)
-        assert re.search(regex, result.stdout) is not None
+        assert re.search(regex, result.stdout) is not None, (
```

Comment:
Needed some additional debug output for this failure

Reply:
The fact that `re.*` return `None` for a missing match means that there's no good way to improve this assertion output generically, unfortunately; there's no way to infer the regex or the text it's matching against in `pytest_assertrepr_compare`.